docs/sources/flow/reference/components/otelcol.exporter.otlphttp.md

```diff
@@ -0,0 +1,205 @@
+`server_name` | `string` | Verifies the hostname of server certificates when set. | | no
+
+The `tls` block should always be provided, even if the server doesn't support
+TLS. To disable `tls` for connections to the HTTP server, set the `insecure`
+argument to `true`.
+
+### queue block
+
+The `queue` block configures an in-memory buffer of batches before data is sent
+to the HTTP server.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`enabled` | `boolean` | Enables an in-memory buffer before sending data to the client. | `true` | no
+`num_consumers` | `number` | Number of readers to send batches written to the queue in parallel. | `10` | no
+`queue_size` | `number` | Maximum number of unwritten batches allowed in the queue at once. | `5000` | no
+
+When `enabled` is `true`, data is first written to an in-memory buffer before
+sending it to the configured HTTP server. Batches sent to the component's
+`input` exported field are added to the buffer as long as the number of unsent
+batches does not exceed the configured `queue_size`.
+
+`queue_size` is used to determine how long an endpoint outage is tolerated for.
+Assuming 100 requests/second, the default queue size `5000` provides about 50
+seconds of outage tolerance. To calculate the correct value for `queue_size`,
+multiply the average number of outgoing requests per second by the amount of
+time in seconds outages should be tolerated for.
+
+The `num_consumers` argument controls how many readers read from the buffer and
+send data in parallel. Larger values of `num_consumers` allow data to be sent
+more quickly at the expense of increased network traffic.
+
+### retry block
+
+The `retry` block configures how failed requests to the HTTP server are
+retried.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`enabled` | `boolean` | Enables retrying failed requests. | `true` | no
+`initial_interval` | `duration` | Initial time to wait before retrying a failed request. | `"5s"` | no
+`max_interval` | `duration` | Maximum time to wait between retries. | `"30s"` | no
+`max_elapsed_time` | `duration` | Maximum amount of time to wait before discarding a failed batch. | `"5m"` | no
+
+When `enabled` is `true`, failed batches are retried after a given interval.
+The `initial_interval` argument specifies how long to wait before the first
+retry attempt. If requests continue to fail, the time to wait before retrying
+increases exponentially. The `max_interval` argument specifies the upper bound
+of how long to wait between retries.
+
+If a batch has not sent successfully, it is discarded after the time specified
+by `max_elapsed_time` elapses. If `max_elapsed_time` is set to `"0s"`, failed
+requests are retried forever until they succeed.
```

Comment:
It looks like both of these blocks can be introduced as new docs snippets since neither this nor otelcol.exporter.otlp introduce component-specific defaults here.